Framework/DataHandling/inc/MantidDataHandling/LoadMuonStrategy.h

```diff
@@ -0,0 +1,50 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+#include "MantidAPI/Workspace_fwd.h"
+#include "MantidDataObjects/TableWorkspace.h"
+#include "MantidDataObjects/Workspace2D.h"
```

Comment:
Can we reduce this to `#include "MantidAPI/Workspace_fwd.h"` instead? Transitive header includes are problem for build times as they get included in other files and have to be read multiple times when included by other files.

Reply:
Yep good point, I can't see any TableWorkspace forward declaration, should I create one myself?

Reply:
It looks like the _sptr declarations are in TableWorkspace https://github.com/mantidproject/mantid/blob/master/Framework/DataObjects/inc/MantidDataObjects/TableWorkspace.h#L432. It would be good to have a `_fwd` file with those in separately. :+1:

Reply:
Okay, I've created a _fwd declaration header for TableWorkspace and Workspace2D.